code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,10 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(is_species(H, /datum/species/squid))
+		H.visible_message("<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
```

Comment:
You don't actually check if it was ingested. Add `if(method == INGEST)`. I recommend adding a slightly different effect for `else if(method == TOUCH)`.

Reply:
Done